app/src/main/java/com/example/footprnt/Map/Util/MapConstants.java

```diff
@@ -25,17 +22,18 @@
     public static int POST_RADIUS = 5;
 
     // Map Styles:
-    public static String map_style = "map_style";
-    public static int style_aubergine = R.raw.style_json_aubergine;
-    public static int style_darkmode = R.raw.style_json_darkmode;
-    public static int style_silver = R.raw.style_json_silver;
-    public static int style_retro = R.raw.style_json_retro;
-    public static int style_basic = R.raw.style_json_basic;
+    public static String MAP_STYLE = "map_style";
+    public static int STYLE_AUBERGINE = R.raw.style_json_aubergine;
```

Comment:
I emphasized creating constants to make things cleaner and easier to understand and change but there is a limit. In Android everything that is R.something is basically a static value itself. We don't need to alias static values to other static values when they're defined elsewhere. A good rule of thumb is that if you want to tweak the value do you need to dig in this file to change it? Then it should likely be a constant, otherwise you can just use it inline. In this case the actual style is defined elsewhere so just use R.raw.style_json_aubergine where you need it.